bitnami/redis/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: redis
-version: 10.7.1
+version: 10.7.2
```

Comment:
I would bump the minor version for this (-> 10.8.0) since it is making important changes to the functionality of the chart.